holoviews/plotting/util.py

```diff
@@ -72,6 +73,65 @@ def collate(obj):
         raise Exception(undisplayable_info(obj))
 
 
+def isoverlay_fn(obj):
+    """
+    Determines whether object is a DynamicMap returning (Nd)Overlay types.
+    """
+    return isinstance(obj, DynamicMap) and (isinstance(obj.last, CompositeOverlay))
+
+
+def linked_zorders(obj, path=[]):
```

Comment:
Although *much* better than before, I'm still not sure this name is right. The ``linked`` part of the name is about how it is used (candidate sources for streams) *not* what it does which is better described by the first part of the docstring which talks about 'objects'.

How about ``object_zorders``?